config/routes.rb

```diff
@@ -55,11 +55,16 @@
         path_prefix: :user
       })
 
-      devise_scope :spree_user do
+      devise_scope :admin_spree_user do
```

Comment:
I'm asking myself if we also need to change the `devise_for` above to `admin_spree_user`?